js/components/discussions/threads.vue

```diff
@@ -123,6 +195,19 @@ export default {
             return this.$refs.threads.find($thread => $thread.discussion.id == id);
         },
 
+        /**
+         * Sort threads by creation date or by last response date
+         */
+
+        sortBy(key){
+
+            if (key == 'created') {
+                this.discussions.sort((a,b) => a['created'] < b['created']);
+            } else if ( key== 'response'){
+                this.discussions.sort((a,b) => a.discussion.slice(-1)[0]['posted_on'] <  b.discussion.slice(-1)[0]['posted_on'] );
```

Comment:
`] )`

Reply:
Not sure `sort` works as expected, cf https://github.com/opendatateam/udata/pull/1642/files